src/pages/IndividualReport.vue

```diff
@@ -0,0 +1,410 @@
+    </PvAccordion>
+  </div>
+</template>
+
+<script setup>
+import { fetchDocById } from '../helpers/query/utils';
+import { runPageFetcher } from '../helpers/query/runs';
+import { useQuery } from '@tanstack/vue-query';
+import { computed, onMounted, ref } from 'vue';
+import { storeToRefs } from 'pinia';
+import { useAuthStore } from '../store/auth';
+import { taskDisplayNames } from '@/helpers/reports';
+import IndividualScoreReportTask from '../components/reports/IndividualScoreReportTask.vue';
+import AppSpinner from '../components/AppSpinner.vue';
+import { getGrade } from '@bdelab/roar-utils';
+import NextSteps from '@/assets/NextSteps.pdf';
+import html2canvas from 'html2canvas';
+import jsPDF from 'jspdf';
+
+// const administrationId = '5vaxicYXnpsNXeq1mUJK';
+// const userId = '00w7xNIlq9gG1uxhmRTiv9NdOys2';
+
+const authStore = useAuthStore();
+
+const { roarfirekit } = storeToRefs(authStore);
+
+const props = defineProps({
+  administrationId: {
+    type: String,
+    required: true,
+  },
+  userId: {
+    type: String,
+    required: true,
+  },
+});
+
+const initialized = ref(false);
+
+const { data: studentData } = useQuery({
+  queryKey: ['users', props.userId],
+  queryFn: () => fetchDocById('users', props.userId),
+  enabled: initialized,
+  keepPreviousData: true,
+  staleTime: 5 * 60 * 1000,
+});
+
+// const {data: classData } = useQuery({
+//   queryKey: ['classes', props.userId],
+//   queryFn: () => fetchDocById('classes', studentData.value.classes.current[0]),
+//   enabled: studentData.value !== undefined,
+//   keepPreviousData: true,
+//   staleTime: 5 * 60 * 1000,
+// })
+
+const { data: taskData } = useQuery({
+  queryKey: ['runs', props.administrationId, props.userId],
+  queryFn: () =>
+    runPageFetcher({
+      administrationId: props.administrationId,
+      userId: props.userId,
+      select: ['scores.computed.composite', 'taskId'],
+      scoreKey: 'scores.computed.composite',
+      paginate: false,
+    }),
+  enabled: initialized,
+  keepPreviousData: true,
+  staleTime: 5 * 60 * 1000,
+});
+
+const { data: rawTaskData } = useQuery({
+  queryKey: ['runs', props.administrationId, props.userId],
+  queryFn: () =>
+    runPageFetcher({
+      administrationId: props.administrationId,
+      userId: props.userId,
+      select: ['scores.computed', 'taskId'],
+      scoreKey: 'scores.computed',
+      paginate: false,
+    }),
+  enabled: initialized,
+  keepPreviousData: true,
+  staleTime: 5 * 60 * 1000,
+});
+
+const { data: administrationData } = useQuery({
+  queryKey: ['administrations', props.administrationId],
+  queryFn: () => fetchDocById('administrations', props.administrationId),
+  enabled: initialized,
+  keepPreviousData: true,
+  staleTime: 5 * 60 * 1000,
+});
+
+const expanded = ref(false);
+const exportLoading = ref(false);
+
+const setExpand = () => {
+  console.log('expand called');
+  expanded.value = !expanded.value;
+};
+
+const IndividualReportDownload = ref(null);
```

Comment:
Style nitpick: recommend camel case with lowercase first character
```suggestion
const individualReportDownload = ref(null);
```